### PR TITLE
[WFCORE-952] Use standard null checks

### PR DESCRIPTION
**Pull request by dmlloyd**

### CODE REVIEW

**Comment by dmlloyd** [comment]
https://issues.jboss.org/browse/WFCORE-952

**Comment by dmlloyd** [comment]
It's retesting, not sure why it's showing failure still.  Whatever

**Comment by bstansberry** [comment]
Looks like modules are missing dependencies on org.wildfly.common.

**Comment by dmlloyd** [comment]
hmm it definitely *was* but it should be fixed in my updated commits, and it seemed to be working locally.  I'll check it again.

**Comment by dmlloyd** [comment]
Should be OK now.  Sorry about that!

**Comment by dmlloyd** [comment]
Guess I didn't run with -DallTests.

File: host-controller/src/main/java/org/jboss/as/domain/controller/resources/DomainRootDefinition.java [hunk 1]
```diff
@@ -361,7 +361,7 @@ public void initialize(ManagementResourceRegistration resourceRegistration) {
         @Override
         public void validateParameter(String parameterName, ModelNode plan) throws OperationFailedException {
             if(plan == null) {
-                throw new OperationFailedException(DomainControllerLogger.ROOT_LOGGER.nullVar("plan").getLocalizedMessage());
+                throw new OperationFailedException(DomainControllerLogger.ROOT_LOGGER.nullVar("plan"));
```
Comment (bstansberry) [inline host-controller/src/main/java/org/jboss/as/domain/controller/resources/DomainRootDefinition.java]: This should have been an assert in the first place -- https://issues.jboss.org/browse/WFCORE-1769. I'll fix that after I merge this.

**Comment by bstansberry** [comment]
If you plan to squash let me know; otherwise I'll merge this as is in my next batch.

**Pull request merged**